### PR TITLE
Project restructure

### PR DESCRIPTION
**Pull request by hizir-bc**

### CODE REVIEW

**Comment by hizir-bc** [comment]
Documentation is remaining, please don't merge until the doc work is done ❌

File: samples/python/src/test/update/diagnostic_test.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED
+"""
+
+import sys
+import pytest
+sys.path.append("../../main")
+from service import diagnostic
+
+
+def test_diagnostic():
```
Comment (vibhor-bc) [inline samples/python/src/test/update/diagnostic_test.py:12]: This script doesn't tests anything rather just prints the API response. So it makes more sense to move it to utility folder.

**Review by vibhor-bc — CHANGES_REQUESTED** [review]

**Comment by hizir-bc** [comment]
> As DiagnosticCheck.ts is required only for typescript samples so moving it under nodejs folder will make more sense.

done

**Comment by hizir-bc** [comment]
Tech writer review is pending for the readme files

**Comment by hizir-bc** [comment]
Tech writer review is pending ❌

**Comment by hizir-bc** [comment]
All the code changes are in placed, can be reviewed.

**Comment by vibhor-bc** [comment]
Waiting for merge until SDK 2.0.0 release.

**Comment by hizir-bc** [comment]
end-to-end test successful ✅

**Pull request merged**